contents/uix/uix.md

```diff
@@ -45,3 +47,65 @@ Just like Software Development, UX Design is an iterative process. It involves m
 ## Which is more important?
 
 The metaphor of building a house by Clayton Yan (UX Designer at UserTesting) illustrates this aptly. UX is the structure (i.e. the number of rooms, one-storey or two-storey and where the front door leads). Meanwhile, UI is about the visual (i.e. whether the same wallpaper or vases is used). A good UX with bad UI is like an ugly house with nice structure. A good UI with bad UX is like a beautiful house with the front door leading straight to the bathroom. To sum it up, UI and UX are **equally** important in creating an awesome experience for the users. 
+
+## The Process of UX Design
+
+As stated above, UX is a highly iterative process to find the right balance for the target audience. Many times the functionality of a product is prioritised over the experience it gives, however this should not be the case when experience is the deciding factor in delighting or frustrating users.
```

Comment:
"UX is a highly iterative process to find the right balance for the target audience" -- balance between what and what?

Grammar nit: fullstop before "however"